### PR TITLE
Document origin detection over UDS on EKS Fargate

### PR DESCRIPTION
**Pull request by ahmed-mez**
### What does this PR do?

Document origin detection over UDS on EKS Fargate

### Preview

https://docs-staging.datadoghq.com/ahmed/uds-origin-eks-fargate/developers/dogstatsd/unix_socket

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by ahmed-mez** [comment]
question for the reviewers: should we add a separate tab `EKS Fargate` instead of adding notes in the Kubernetes tab?

File: content/en/developers/dogstatsd/unix_socket.md [hunk 1]
```diff
@@ -103,6 +103,24 @@ To enable the Agent DogStatsD UDS:
 
         **Note**: Remove `readOnly: true` if your application containers need write access to the socket.
 
+**Note**: EKS Fargate doesn't support mounting host paths. Alternatively, the socket can be shared between the agent sidecar container and the application container using a `emptyDir` volume.
+    - Mount the socket folder in your `datadog-agent` container and the application container in the pod:
```
Comment (clamoriniere) [inline content/en/developers/dogstatsd/unix_socket.md:107]: you might split it in two items

* Create the volume  (emptyDir) in the applicative pod.
* Mount the volume in each pod's containers

**Comment by kayayarai** [comment]
> should we add a separate tab EKS Fargate instead of adding notes in the Kubernetes tab?

@ahmed-mez Hi! I think yes, we should do that. Go ahead, plus address Cedric's comments, and the docs team will review when that's ready.

File: content/en/developers/dogstatsd/unix_socket.md [hunk 2]
```diff
@@ -192,6 +210,15 @@ When running inside a container, DogStatsD needs to run in the host's PID namesp
         hostPID: true
     ```
 
+**Note**: EKS Fargate doesn't support `hostPID`. Alternatively, you can set `shareProcessNamespace: true` in the pod template spec.
```
[the remark this answers is not in the document]
Comment (kayayarai) [reply content/en/developers/dogstatsd/unix_socket.md:213]:
```suggestion
**Note**: EKS Fargate doesn't support `hostPID`, so instead set `shareProcessNamespace: true` in the pod template spec to be able to use UDS origin detection.
```

**Comment by ahmed-mez** [comment]
@cswatt thanks for the approval. I'm not allowed to merge the PR (is it a new policy?), could you merge it please? Thanks!

**Pull request merged**